### DISCUSSION TITLE
Issue while creating the flux image repository for the local private registry running at `controlplane:5000`

### DISCUSSION THREAD

**Discussion by karnatisrinivas** [fluxcd/flux2#4206] [Q&A]
Hello All and goodday,
I am trying to create a flux image repository for my private local registry running at `controlplane:5000` . I am following the instructions from the documentation:https://fluxcd.io/flux/cmd/flux_create_image_repository/ . 

I tried setting up an image pull secret and referring that, it did not worked. 
Also I have tried creating the tls secret from 'crt' and 'key' files and using the secret in flux create image repository command. Also tried with .pem way as well.

Is there any more documentation for this, or is this still in development? If anyone has any input, it will be of much help.
Thanks.

`Warning  AuthenticationFailed  50s (x11 over 14m)  image-reflector-controller  failed to configure authentication options: unknown secret type "Opaque"`

`scan failed: Get "https://controlplane:5000/v2/": tls: failed to verify certificate: x509: certificate signed by unknown authority`

**Comment by stefanprodan** [comment]
Private registries with self-signed certs are fully supported in Flux 2.1.0, please see here the docs: https://fluxcd.io/flux/components/image/imagerepositories/#certificate-secret-reference

**Reply by sysarcher** [reply]
Hey @stefanprodan is it possible to use an "insecure" registry? i.e. not specifying the cert?
